### PR TITLE
[refactor] Use three single quotes for docstrings

### PR DESCRIPTION
**Pull request by vkarak**
- For consistency with the single quotes used for strings in general.

### CODE REVIEW

File: reframe/core/deferrable.py [hunk 1]
```diff
@@ -57,19 +57,19 @@ def evaluate(self):
             return ret
 
     def __bool__(self):
-        """The truthy value of a deferred expression.
+        '''The truthy value of a deferred expression.
 
         This causes the immediate evaulation of the deferred expression.
```
Comment (teojgo) [inline reframe/core/deferrable.py]: evaulation -> evaluation

File: reframe/utility/__init__.py [hunk 2]
```diff
@@ -100,9 +100,9 @@ def decamelize(s, delim='_'):
 
 
 def toalphanum(s):
-    """Convert string ``s`` be replacing any non-alphanumeric character with
+    '''Convert string ``s`` be replacing any non-alphanumeric character with
```
Comment (teojgo) [inline reframe/utility/__init__.py]: be -> by

File: reframe/utility/os_ext.py [hunk 3]
```diff
@@ -286,13 +286,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 def is_url(s):
-    """Check if string is an URL."""
+    '''Check if string is an URL.'''
```
Comment (teojgo) [inline reframe/utility/os_ext.py]: an -> a

File: reframe/utility/os_ext.py [hunk 4]
```diff
@@ -286,13 +286,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
     parsed = urlparse(s)
     return parsed.scheme != '' and parsed.netloc != ''
 
 
 def git_clone(url, targetdir=None):
-    """Clone git repository from an URL."""
+    '''Clone git repository from an URL.'''
```
Comment (teojgo) [inline reframe/utility/os_ext.py]: an -> a

File: reframe/core/pipeline.py [hunk 5]
```diff
@@ -725,78 +725,78 @@ def perfvalues(self):
...
 
         :type: :class:`reframe.core.schedulers.Job`.
-        """
+        '''
         return self._job
 
     @property
     def logger(self):
-        """A logger associated with the this test.
+        '''A logger associated with the this test.
```
Comment (teojgo) [inline reframe/core/pipeline.py]: the this -> this

File: reframe/core/pipeline.py [hunk 6]
```diff
@@ -1252,7 +1252,7 @@ def _copy_job_files(self, job, dst):
         shutil.copy(script, dst)
 
     def _copy_to_outputdir(self):
-        """Copy checks interesting files to the output directory."""
+        '''Copy checks interesting files to the output directory.'''
```
Comment (teojgo) [inline reframe/core/pipeline.py]: checks -> check's

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**